cdap-docs/developers-manual/source/building-blocks/workflows.rst

```diff
@@ -16,49 +16,56 @@ transferred to the next job in sequence until the last job in the sequence is ex
 failure, the execution is stopped at the failed job and no subsequent jobs in the sequence
 are executed.
 
-To process MapReduce or Spark programs in sequence, specify
-``addWorkflow()`` in your application::
+To execute MapReduce or Spark programs in a Workflow, you will need to add them in your
+application along with the Workflow. You can optionally add a schedule to the Workflow::
```

Comment:
You can optionally add a schedule (such as a `crontab schedule <../../reference-manual/javadocs/co/cask/cdap/api/app/AbstractApplication.html#scheduleWorkflow(java.lang.String,%20java.lang.String,%20java.lang.String)>`__) to the Workflow::